browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -229,8 +230,12 @@ class MarkdownNoteDetail extends React.Component {
     if (e.keyCode === 27) this.handleDeleteCancelButtonClick(e)
   }
 
-  handleShowLockButton () {
-    this.setState({editorStatus: this.refs.content.state.status})
+  handleToggleLockButton () {
+    if (this.props.config.editor.switchPreview === 'BLUR' && this.refs.content.state.status === 'CODE') {
```

Comment:
This is a bug fix for a bug which handles with switching preview by right click.